### PR TITLE
Fix typos. Make sentences make sense, add links for linting, add Pyth…

### PR DESCRIPTION
**Pull request by allthatilk**
…on to linting title for clarity

### CODE REVIEW

File: contribute.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Code Conventions
 It is important to remember that IATI is still a relatively young project and so our current code base has varying standards of code quality and style. We are continuously working towards improving this situation. With this in mind, we ask that you use our `pyIATI <https://github.com/IATI/pyIATI>`__ work as the expected standard to work towards.
 
 `pyIATI is part of our current work to improve the stability and maintainability of our
-infrastructure <https://discuss.iatistandard.org/t/introducing-the-iati-python-library/720>`__ and we believe it exemplifies our move towards stronger architecture and adoption of industry-standard best practices wherever possible and be consistent with the coding style used with this project whenever possible.
+infrastructure <https://discuss.iatistandard.org/t/introducing-the-iati-python-library/720>`__. We believe it exemplifies our move towards stronger architecture and adoption of industry-standard best practices. Because of this we wish to be consistent with the coding style used with throughout this project whenever possible.
```
Comment (amy-silcock) [inline contribute.rst]: Because of this, we wish to be consistent with the coding style used throughout this project wherever possible.

File: contribute.rst [hunk 2]
```diff
@@ -28,34 +28,34 @@ Generally speaking we expect all contributions to adhere to the following conven
...
+   -  All core functionality is unit tested with `py.test <https://docs.pytest.org/en/latest/>`__ and edge cases considered. We strongly suggest you use Test Driven Development. We also require details and evidence of any manual testing to show no existing functionality is unexpectedly broken.
 
 -  Documented code:
 
-   -  We expect a clear docstring per module, class, and function, explaining what it does at a minimum. This is to reduce developer onboarding time and reduce barriers to entry for new developers who want to get involved. They should be formatted as per `Google-style Sphinx Docstrings <http://www.sphinx-doc.org/en/stable/ext/example_google.html>`__.
+   -  We expect a clear docstring per module, class, and function, explaining what it does at a minimum. This is to reduce developer onboarding time and the barriers to entry for new developers who want to get involved. They should be formatted as per `Google-style Sphinx Docstrings <http://www.sphinx-doc.org/en/stable/ext/example_google.html>`__.
```
Comment (amy-silcock) [inline contribute.rst]: This is to reduce both developer onboarding time and the barriers to entry for new developers who want to get involved.

File: contribute.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@ Contribution Guide
...
 
-Please follow this guide if you wish your Pull Requests to be reviewed by us. This guide was created with to ensure your contributions have a meaningful long-term impact on the IATI code base by ensuring your code can be maintained over the long-term. It is also beneficial to open a Github Issue about any significant features you may wish to add as we reserve the right to reject pull requests that clash with, or otherwise hinder, existing or ongoing work.
+Please follow this guide if you wish your pull requests to be reviewed by us. This guide was created to ensure your contributions have a meaningful long-term impact on the IATI code base by ensuring your code can be maintained over the long-term. It is also beneficial to open a Github Issue about any significant features you may wish to add as we reserve the right to reject pull requests that clash with, or otherwise hinder, existing or ongoing work.
 
 This style guide is inspired by the `React contribution guides <https://facebook.github.io/react-native/docs/contributing.html#how-to-contribute>`__.
```
Comment (hayfield) [inline contribute.rst]: `style guide` -> `guide`

(a style guide is something different)

File: contribute.rst [hunk 4]
```diff
@@ -28,34 +28,34 @@ Generally speaking we expect all contributions to adhere to the following conven
 
 -  Robustly tested code:
 
-   -  All core functionality unit tested with `py.test <https://docs.pytest.org/en/latest/>`__ and edge cases considered. We strongly suggest you use Test Driven Development. We also require details and evidence of any manual testing to show no existing functionality is unexpectedly broken.
+   -  All core functionality is unit tested with `py.test <https://docs.pytest.org/en/latest/>`__ and edge cases considered. We strongly suggest you use Test Driven Development. We also require details and evidence of any manual testing to show no existing functionality is unexpectedly broken.
```
Comment (hayfield) [inline contribute.rst]: `is` -> `is to be`

(`is` isn't in the same tense as the surrounding content)

File: contribute.rst [hunk 5]
```diff
@@ -28,34 +28,34 @@ Generally speaking we expect all contributions to adhere to the following conven
...
 
 -  No code smells:
 
    -  Don’t Repeat Yourself, keep your code DRY.
 
-   -  Avoid excessive use of conditional statements, your functions should be doing the minimum possible for maximum effect. Think about `polymorphism <https://www.digitalocean.com/community/tutorials/how-to-apply-polymorphism-to-classes-in-python-3>`__ and the `Single Responsibility Princible (SRP) <https://robots.thoughtbot.com/back-to-basics-solid#single-responsibility-principle>`__.
+   -  Avoid excessive use of conditional statements, your functions should be doing the minimum possible for maximum effect. Think about `polymorphism <https://www.digitalocean.com/community/tutorials/how-to-apply-polymorphism-to-classes-in-python-3>`__ and the `Single Responsibility Principle (SRP) <https://robots.thoughtbot.com/back-to-basics-solid#single-responsibility-principle>`__.
```
Comment (hayfield) [inline contribute.rst]: A comma doesn't seem like quite the right punctuation in the middle of this sentence.
